.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: WikibaseReconcileEdit CI
+
+on:
+    push:
```

Comment:
IIRC this should also include `pr:`, to run CI on pull requests that don’t come from this repo?

Reply:
done